tests/unit_tests/test_deplete_operator.py

```diff
@@ -0,0 +1,72 @@
+"""Basic unit tests for openmc.deplete.Operator instantiation
+
+Modifies and resets environment variable OPENMC_CROSS_SECTIONS
+to a custom file with new depletion_chain node
+"""
+
+from os import remove
+from os import environ
+from unittest import mock
+from pathlib import Path
+import pytest
+
+from openmc.deplete.abc import TransportOperator
+from openmc.deplete.chain import Chain
+
+BARE_XS_FILE = "bare_cross_sections.xml"
+CHAIN_PATH = Path().cwd() / "tests" / "chain_simple.xml"
```

Comment:
This only works if the current working directory is correct. I believe you could use `Path(__file__).parents[1] / "chain_simple.xml"` for it to work anywhere.